client/app/host/TeamInspector.tsx

```diff
@@ -0,0 +1,87 @@
+'use client';
+
+import {
+    Select,
+    SelectContent,
+    SelectItem,
+    SelectTrigger,
+    SelectValue,
+} from '@/components/ui/select';
+import { ArrowLeft, Wifi, WifiOff } from 'lucide-react';
+import { useTeams, useSelectedTeam } from '@/lib/host-state';
+import { Separator } from '@/components/ui/separator';
+import { Card, CardHeader, CardTitle } from '@/components/ui/card';
+import TeamInfo from './TeamInfo';
+import { Button } from '@/components/ui/button';
+
+export default function TeamInspector() {
+    const { teamList } = useTeams();
+    const { selectedTeam, setSelectedTeam } = useSelectedTeam();
+
+    return (
+        <div className="flex flex-col">
+            <div
+                className={`flex h-full w-full flex-row items-center ${selectedTeam === null ? 'justify-end' : 'justify-between'}`}
+            >
+                {selectedTeam !== null && (
+                    <Button variant="ghost" className="flex" onClick={() => setSelectedTeam(null)}>
+                        <ArrowLeft />
+                        View All Teams
+                    </Button>
+                )}
+                <Select
+                    value={selectedTeam?.name || ''}
+                    onValueChange={(value) => setSelectedTeam(teamList[+value])}
+                >
+                    <SelectTrigger className="flex w-fit">
+                        <SelectValue placeholder="Select A Team" />
+                    </SelectTrigger>
+                    <SelectContent>
+                        {teamList.map((team, index) => (
+                            <SelectItem value={`${index}`} key={index}>
+                                <span className="flex gap-1">
+                                    {team.status ? (
+                                        <Wifi className="text-green-500" />
+                                    ) : (
+                                        <WifiOff className="text-gray-300 dark:text-gray-500" />
+                                    )}
+                                    {team.name}
+                                </span>
+                            </SelectItem>
+                        ))}
+                    </SelectContent>
+                </Select>
```

Comment:
When a team is selected, this is blank.  It should show the selected team
![screenshot](https://github.com/user-attachments/assets/04c6a0ce-2ef5-4b3c-abc1-fb91bf8d3886)

(It may be because of the `value` of the Select component)